neo/test/iotest/test_neuralynxio.py

```diff
@@ -311,7 +311,7 @@ def _load_plaindata(self, filename, numSamps):
             #     np.testing.assert_allclose(plain_data[:numToTest],
             #                                anasig.magnitude[:numToTest, 0] * gain_factor_0,
             #                                rtol=0.01, err_msg=" for file " + filename)
-    @unittest.skip
+    @unittest.skip("nse failing for now as per issue #907")
```

Comment:
are you sure this one is still failing? At least locally it succeeds for me.

Reply:
Also possibly unintentionally included, but works fine.